dialogue-annotations/src/main/java/com/palantir/dialogue/annotations/ConjureErrorDecoder.java

```diff
@@ -18,6 +18,7 @@
 
 import com.palantir.dialogue.Response;
 
+// TODO(pm): use the new EndpointErrorDecoder
```

Comment:
Replace impl in `ErrorDecoder` to delegate to the empty-map initialized EndpointErrorDecoder